src/components/job-board/jobBoard.tsx

```diff
@@ -14,73 +17,139 @@ export default function JobBoard() {
+                  </div>
+                </div>
+
+                <div className="flex flex-wrap gap-2">
                   {selectedJob.tags.map((tag) => (
                     <span
                       key={tag.id}
-                      className="rounded-lg bg-green-100 px-2 py-1 text-sm font-medium text-green-600"
+                      className={`rounded-full bg-[#FFF5F3] px-4 py-1.5 text-sm font-medium text-[#FF5331] border border-[#FF5331]`}
                     >
                       {tag.name}
                     </span>
                   ))}
                 </div>
-                <p className="mt-4 text-gray-700">{selectedJob.description}</p>
-              </>
+
+                <div className="space-y-4">
+                  <h2 className="text-2xl font-semibold text-gray-900">Job Description</h2>
+                  <p className="text-gray-700 leading-relaxed">{selectedJob.description}</p>
+                </div>
+
+                <div className="flex flex-col gap-3">
+                  <div className="flex items-center gap-3">
+                    <Button className="h-10 bg-[#FF5331] px-6 hover:bg-[#E64A2A]">
+                      <CheckCircle className="mr-2 h-4 w-4" />
+                      Apply Now
+                    </Button>
+                    <Button variant="outline" size="icon" className="h-10 w-10 border-[#FF5331] text-[#FF5331] hover:bg-[#FFF5F3]">
+                      <BookmarkPlus className="h-5 w-5" />
+                    </Button>
+                    <Button variant="outline" size="icon" className="h-10 w-10 border-[#FF5331] text-[#FF5331] hover:bg-[#FFF5F3]">
+                      <Share2 className="h-5 w-5" />
+                    </Button>
+                  </div>
+                  <div className="flex items-center justify-between border-t border-gray-100 pt-3">
+                    <a
+                      href={selectedJob.url}
+                      target="_blank"
+                      rel="noopener noreferrer"
+                      className="text-sm font-medium text-[#FF5331] hover:underline"
+                    >
+                      View original job posting →
+                    </a>
+                    <div className="flex gap-2">
+                      <Button variant="ghost" size="sm" className="h-8 px-2 text-gray-500 hover:bg-gray-100">
+                        <Flag className="h-4 w-4" />
+                      </Button>
+                    </div>
+                  </div>
+                </div>
+              </div>
             ) : (
-              <h5 className="text-gray-500">Select a job to see details</h5>
+              <div className="flex h-full items-center justify-center">
+                <h5 className="text-lg text-gray-500">Select a job to see details</h5>
+              </div>
             )}
-          </div>
+          </ScrollArea>
         </div>
     );
-  }
\ No newline at end of file
+}
```

Comment:
We should always have a newline at the end of every file. That's why Github puts this red icon here.